src/nvim/shada.c

```diff
@@ -2318,6 +2318,55 @@ static inline ShaDaWriteResult shada_read_when_writing(
   return ret;
 }
 
+/// Get list of buffers to write to the shada file
+///
+/// @param[in] khash_t(bufset)   Buffers which are ignored
```

Comment:
`removable_bufs`, type is not what should be put here. Also two spaces before variable name and two spaces after.